src/LoginServer/LoginServer.cs

```diff
@@ -54,7 +54,11 @@ public override void Run()
 			this.CheckDatabaseUpdates();
 
 			// Data
-			this.LoadData(DataToLoad.All, true);
+			this.LoadData(DataToLoad.Jobs, true);
```

Comment:
You could also do this: `this.LoadData(DataToLoad.Jobs | DataToLoad.Maps | DataToLoad.Barracks | [...], true);`